src/FixMixedTabs/Impl/MarginFactory.cs

```diff
@@ -18,16 +18,16 @@ namespace FixMixedTabs
     internal sealed class MarginFactory : IWpfTextViewMarginProvider
     {
         [Import]
-        private ITextDocumentFactoryService _TextDocumentFactoryService = null;
-        [Import]
+        private ITextDocumentFactoryService _TextDocumentFactoryService;
```

Comment:
No, project cannot be built on my PC too. Anyway, I return these null assignments back.